React/finger-exercises/react/ecmascript/02 - promises/index.js

```diff
@@ -1,9 +1,20 @@
 // Hint: use setInterval, create a new Promise and measure time with Date.now()
 
-export function delay() {
-
+export function delay(time) {
+  const startTime = Date.now();
+  return new Promise((resolve, reject) => {
+    setInterval(function() {
+      const endTime = Date.now();
+      const timeRandom = time - (endTime - startTime);
+      if ((timeRandom) > (time + 100)) {
+        reject(Error('This time is too much !'));
+      } else {
+        resolve(timeRandom);
+      }
+    }, 50);
+  });
 }
 
-export function asyncDelay() {
-
+export function asyncDelay(time) {
```

Comment:
the `async/await` syntax is missing... you can find more information on how to use it here: 
https://javascript.info/async-await

Reply:
the syntax is already being tested, so I only use the function

![image](https://user-images.githubusercontent.com/25165141/46488621-d2462880-c7c8-11e8-83da-41049eaaa4ee.png)